docs/virtual_machines/disks_and_volumes.md

```diff
@@ -1257,3 +1257,150 @@ Example: force `writethrough` cache mode
         persistentVolumeClaim:
           claimName: disk-alpine
     status: {}
+
+### Disk sharing
+
+Shareable disks allows multiple VMs to share the same underlying storage. In order to use this feature, a special care is required because this could lead to data corruption and the loss of important data. Shareable disks demand either data synchronization at application level or the usage of clustered filesystems. These advanced configurations are out-of-scope of this documentation and use-cases specific.
```

Comment:
a special care -> special care
Shareable disks demand either data synchronization at application level or the usage of clustered filesystems -> Shareable disks demand either data synchronization at the application level or the use of clustered filesystems.
..are out-of-scope of this documentation and use-cases specific -> are not within the scope of this documentation and  are use-case specific.